lib/rust/ensogl/core/src/display/scene/layer.rs

```diff
@@ -886,6 +908,37 @@ impl SublayersModel {
 
 
 
+// ===================
+// === MaskedLayer ===
+// ===================
+
+/// A layer with attached mask. Every object in `mask` layer will mask the objects of the
+/// `masked_object` layer.
```

Comment:
That's not true. If the objects in the mask layer will be rendered as black, they will not mask the objects in the base layer. (ofc. depending how you define "masking". If you define it as "removing", then objects rendered in white would not work as described here).

Reply:
I rephrased the docs. Your explanation is also not true. We don't use black/white masks, we use opaque/transparent instead (we only look at `alpha` channel).